tina/config.ts

```diff
@@ -166,6 +166,17 @@ export default defineConfig({
               component: "tags",
             },
           },
+          {
+            type: "string",
+            name: "ProjectCategory",
+            label: "Project Category",
+            list: true,
+            required: false,
+            ui: {
+              component: "categories",
+            },
+          },
```

Comment:
yeah, I can't change this, because these are the meta data that will be placed in the markdown file, So tina CMS will create a CMS based on the config file tile and the name need to be same as the meta data that you  are pointing to, so it can't be changed.